test/e2e/specs/specs-playwright/wp-auth__canary-spec.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import assert from 'assert';
+import config from 'config';
+import { DataHelper, BrowserHelper, LoginFlow } from '@automattic/calypso-e2e';
+
+/**
+ * Constants
+ */
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const host = DataHelper.getJetpackHost();
+const viewportName = BrowserHelper.getViewportName();
+
+describe( `[${ host }] Authentication: (${ viewportName }) @canary @parallel @safaricanary`, function () {
+	this.timeout( mochaTimeOut );
+
+	it( 'User agent is set', async function () {
```

Comment:
Not for this PR... but I think we should eventually remove this test.

It is not testing the product, it is testing that e2e has been set up correctly. In other words, this is an unit test for the project in `test/e2e`.